tools/templates/vfvs_run.py

```diff
@@ -2152,6 +2185,26 @@ def docking_finish_vina(item, ret):
         matches = match.groupdict()
         item['score'] = float(matches['value'])
         item['status'] = "success"
+
+        if(item['run_pose_check'] == 1):
+
+            # Load in config file:
+            with open(item['config_path']) as fd:
+                config_ = dict(read_config_line(line) for line in fd)
+            for element in config_:
+                if '#' in config_[element]:
+                    config_[element] = config_[element].split('#')[0]
+
+            # Convert docking output to sdf (first model only)
+            sdf_file = item['output_path'] + '.sdf'
+            subprocess.run(['obabel', '-ipdbqt', item['output_path'], '-f', '1', '-l', '1', '-osdf', '-O', sdf_file])
+            pdb_file = os.path.join(item['input_files_dir'], config_['receptor'].replace('pdbqt', 'pdb'))
+
+            # Run pose check
+            if(posebusters_check(item, sdf_file, pdb_file, item['pose_check_timeout']) != 1):
+                item['score'] = None
+                item['status'] = "failed"
```

Comment:
either set the item['log']['reason'] here with why it failed, or do it in the function. It looks like it only gets set in the function currently if it times out